youtube_dl/YoutubeDL.py

```diff
@@ -563,7 +563,7 @@ def prepare_filename(self, info_dict):
             if autonumber_size is None:
                 autonumber_size = 5
             autonumber_templ = '%0' + str(autonumber_size) + 'd'
-            template_dict['autonumber'] = autonumber_templ % self._num_downloads
+            template_dict['autonumber'] = autonumber_templ % (self.params.get('autonumber_start', 1) - 1 + self._num_downloads)
```

Comment:
if I make `self._num_downloads` start from something other then 0, wouldn't that break this code ? https://github.com/costypetrisor/youtube-dl/blob/autonumber_start/youtube_dl/YoutubeDL.py#L1439

To make it would I would have to change both the initialization of `self._num_downloads` and this if that checks against `max_downloads`